src/main/java/me/moodcat/backend/RoomBackend.java

```diff
@@ -226,7 +231,6 @@ public void sendMessage(final ChatMessage chatMessage) {
          * Merge the changes of the instance in the database.
          */
         protected void merge() {
-            log.info("Merging changes in room {}", this);
```

Comment:
Please keep the log for a while. Maybe decrease the log level to debug so we can filter it in production.